dbt_meshify/main.py

```diff
@@ -51,3 +51,26 @@ def split():
         holder.register_project(subproject)
 
     print(holder.project_map())
+
+
+@cli.command(name="contract")
```

Comment:
Non-blocking feedback: I wonder if this command should be named more descriptively for what it does. For instance, maybe `generate_contracts` or some other phrasing that indicates that contracts are being created?

Reply:
i like that idea! i was optimizing for verbs, and brevity, thinking that some other commands could be `publicize`, `group` and the like, but there are likely limits to that approach

Reply:
Let's roll with it as is, and see how it feels! Definitely not something to bike shed this early 😝